nixos/modules/services/audio/mpd.nix

```diff
@@ -54,6 +54,14 @@ in {
         '';
       };
 
+      startSystemWide = mkOption {
```

Comment:
After some discussion involving `tmpfiles` and the appropriateness of specifying normal users for NixOS modules (which I'm almost certain you were a part of, sorry if I'm mistaken...) I think this option might become a bit of at least a little bit of a pattern going forward. That being said it is always good to have consistent naming conventions. `startSystemWide` is a very clear name so we should all agree to push this name as a loose 'standard' going forward, yeah?

Reply:
> the appropriateness of specifying normal users for NixOS modules

That's not really what this is. This is about making it available in a declarative way to normal users but not forcing it to run.

> should all agree

I'm not sure that all those that should agree are seeing this... ;-)